src/FaveListItem/FaveListItem.js

```diff
@@ -0,0 +1,27 @@
+import React, { Component } from "react";
+import "./FaveListItem.scss";
+
+class FaveListItem extends Component {
+  constructor(props) {
+    super(props);
+  }
+
+  removeFaveListItem = event => {
+    event.preventDefault();
+    this.props.removeFaveListItem(event.target.closest("li").id)
+  }
```

Comment:
This removes elements from the array in App's state, right? Have you tested deleting one element and then deleting another element with an index greater than the first? Wouldn't removing an element in the array change all the indices of the elements after it but each li would keep their old id?

On second thought, since the method calls setState, there would be a re-render and the array would be mapped over again which would result in the correct ids for each li.

Reply:
Oh. I hadn't caught that! It is only deleting the first one. I'll take a look and come up with a way to just remove the targeted item.

Reply:
It wasn't working because I had the id on the i tag and not the li item.  And yes, you are correct that it does work because the method calls setState so the ids re-render each time one is deleted. I fixed it so I will re-push  to this branch.